python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { ClientPrompt } from "../../shared/types";
+import { Button, createStyles, Flex, Text } from "@mantine/core";
+import { IconPlayerPlayFilled } from "@tabler/icons-react";
+import { memo } from "react";
+
+type Props = {
+  prompt: ClientPrompt;
+  size: "compact" | "full";
+};
+
+const useStyles = createStyles(() => ({
+  executeButton: {
+    borderBottomLeftRadius: 0,
+    borderTopLeftRadius: 0,
+    borderTopRightRadius: 0,
```

Comment:
Why are we only defining 3 of the corners? Also is 0 the default and not needed?

Reply:
We want the bottom right to have radius to match the card border. Default is 0.25em, which already matches the prompt card border radius